cli/api.py

```diff
@@ -93,15 +94,18 @@ def generate_api(input_dir, output, summary_output, aggregation_level, state, fi
     us_latest = combined_datasets.load_us_latest_dataset().get_subset(
         aggregation_level, state=state, fips=fips, states=active_states
     )
-    us_timeseries = combined_datasets.load_us_timeseries_dataset().get_subset(
-        aggregation_level, state=state, fips=fips, states=active_states
-    )
+    regions = [
+        pipeline.Region.from_fips(fips) for fips in us_latest.all_fips if not fips.endswith("999")
+    ]
 
     for intervention in list(Intervention):
         _logger.info(f"Running intervention {intervention.name}")
-        all_timeseries = api_pipeline.run_on_all_fips_for_intervention(
-            us_latest, us_timeseries, intervention, input_dir
-        )
+        regional_inputs = [
+            api_pipeline.RegionalInput.from_region_and_intervention(region, intervention, input_dir)
+            for region in regions
+        ]
+        _logger.info(f"Loaded {len(regional_inputs)} regions.")
+        all_timeseries = api_pipeline.run_on_all_regional_inputs_for_intervention(regional_inputs)
```

Comment:
Not sure the right time, but may be eventually nice to run all (region X interventions) at the same time. Or iterate through regions as the outside loop so this can be done as part of the pyseir per-region pipeline. certainly doesn't need to be part of this refactor.